### ISSUE TITLE
Event name: All Things Open 2020

### ISSUE THREAD

**Issue by chrisblumOMP** [openmainframeproject/foundation#117]
Event dates: Oct 13 - 15, 2020	
Location: Raleigh, NC	URL: http://allthingsopen.org/		
	
Speaking Submitted:
- Creating a sustainable open source ecosystem for mainframe through the Open Mainframe Project
- Bridging Modern DevOps and the Mainframe

Speaking Accepted. Waiting on which tracks:
https://2020.allthingsopen.org/speakers/elizabeth-joseph/
https://2020.allthingsopen.org/speakers/john-mertic/

**Comment by jmertic** [comment]
Duplicate of #116
